pyrk/materials/cool.py

```diff
@@ -0,0 +1,55 @@
+from ur import units
+from density_model import DensityModel
+from material import Material
+import random
+
+class Cool(Material):
```

Comment:
Can I just delete all the orig files, safely?

On Tue, Aug 25, 2015 at 11:21 AM, Katy Huff <notifications@github.com>
wrote:

> In pyrk/materials/cool.py
> <https://github.com/pyrk/pyrk/pull/10#discussion_r37900228>:
>
> > @@ -0,0 +1,55 @@
> > +from ur import units
> > +from density_model import DensityModel
> > +from material import Material
> > +import random
> > +
> > +class Cool(Material):
>
> Materials are not meant to be called things like "cool" because many
> different materials (water, sodium, flinak, flibe, helium) can be a reactor
> coolant. You seem to be describing flibe. For this, we already have
> flibe.py. Please delete this file and make necessary changes to flibe.py
> instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyrk/pyrk/pull/10/files#r37900228>.
>



-- 
-----------------------------------------
Xin WANG | 王昕
PhD Student
Thermal Hydraulics Laboratory
Nuclear Engineering Department
University of California, Berkeley
(415)408-1691